src/test/java/com/epam/tc/hw7/BaseTest.java

```diff
@@ -0,0 +1,106 @@
+package com.epam.tc.hw7;
+
+import static com.epam.jdi.light.driver.WebDriverUtils.killAllSeleniumDrivers;
+import static com.epam.jdi.light.elements.init.PageFactory.initSite;
+import static com.epam.jdi.light.settings.WebSettings.logger;
+import static com.epam.tc.hw7.ententies.TopMenuData.MetalsColors;
+import static com.epam.tc.hw7.site.SiteJdi.homePage;
+import static com.epam.tc.hw7.site.SiteJdi.loginForm;
+import static com.epam.tc.hw7.site.SiteJdi.metalsColoursPage;
+import static com.epam.tc.hw7.site.SiteJdi.topMenu;
+import static com.epam.tc.hw7.site.SiteJdi.userIcon;
+import static com.epam.tc.hw7.site.custom.MultiDropdown.colourDropdown;
+import static com.epam.tc.hw7.site.custom.MultiDropdown.metalsDropdown;
+import static com.epam.tc.hw7.site.custom.MultiDropdown.vegetablesDropdown;
+import static com.epam.tc.hw7.site.pages.MetalsColoursPage.elementsCheckList;
+import static com.epam.tc.hw7.site.pages.MetalsColoursPage.evenRadioButtons;
+import static com.epam.tc.hw7.site.pages.MetalsColoursPage.oddRadioButtons;
+import static com.epam.tc.hw7.site.pages.MetalsColoursPage.resultText;
+import static com.epam.tc.hw7.site.pages.MetalsColoursPage.submitButton;
+import static com.epam.tc.hw7.states.States.shouldBeLoggedIn;
+import static com.epam.tc.hw7.states.States.shouldBeLoggedOut;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.tc.hw7.ententies.LoginUserData;
+import com.epam.tc.hw7.site.SiteJdi;
+import io.qameta.allure.Step;
+import org.testng.Assert;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+public class BaseTest {
+
+    @BeforeSuite(alwaysRun = true)
+    public static void setUp() {
+        initSite(SiteJdi.class);
+        logger.info("Run Tests");
+    }
+
+    @AfterSuite(alwaysRun = true)
+    static void teardown() {
+        killAllSeleniumDrivers();
+    }
+
+    @Test
+    public void loginTest() {
+        homePage.isOpened();
+        shouldBeLoggedOut();
+        userIcon.click();
+        loginForm.loginAs(new LoginUserData());
+        homePage.checkOpened();
+    }
+
+    @Test
+    public void metalsColoursPageTest() {
+        shouldBeLoggedIn();
+        topMenu.select(MetalsColors);
+        metalsColoursPage.shouldBeOpened();
+        ReadTestDataFromJson dataFromJson = new ReadTestDataFromJson();
```

Comment:
You need to use @Test(dataProvider = "name") here

Reply:
changed ReadTestDataFromJson to data provider class